src/TimeSpans.jl

```diff
@@ -7,6 +7,8 @@ export TimeSpan, start, stop, istimespan, translate, overlaps,
        time_from_index
 
 
+const NS_IN_SEC = 1_000_000_000  # Number of nanoseconds in one second
```

Comment:
We could do
```suggestion
const NS_IN_SEC = Dates.value(Nanosecond(Second(1)))  # Number of nanoseconds in one second
```
which is the same number aha, in case we trusted Dates.jl more than our own counting